### ISSUE TITLE
Support different groups for different users.

### ISSUE THREAD

**Issue by terjekv** [terjekv/github-authorized-keys#1]
Not everyone needs admin rights, always, for every host supported. So, offering something like:

```shell
GITHUB_ADMIN_TEAM=sshadmins
GITHUB_USER_TEAM=sshusers
```

Would be nice. Ideally, we may also want to support an option to grant admin access to a specific user for a given host (ie, when we establish a temporary node to a user). We could do this via an option ala:

```shell
GITHUB_NON_TEAM_ADMIN=username
GITHUB_NON_TEAM_USER=anotheruser
```

One this we would have to manage is coalescing users who are in multiple groups.

For backwards compatibility the existing `GITHUB_TEAM` should be treated as `GITHUB_ADMIN_TEAM`.

**Comment by terjekv** [comment]
For group variables, we should have something like:

```
SYNC_USERS_ADMIN_GROUPS=wheel
SYNC_USERS_USERS_GROUPS=users,docker
```

Existing group variables (`SYNC_USERS_GROUP` etc) should, for backwards compatilbity, be treated as `SYNC_USERS_ADMIN_GROUP`.

**Comment by terjekv** [comment]
Eh, I might just break backwards compatibility. It's as clean a time to do it as ever. :(

**Comment by terjekv** [comment]
Resolved in 3f7dbe320536c76636525fc2063bb073645f597c.
